### DISCUSSION TITLE
Alternate calendar (e.g. persian) / localization not just translations

### DISCUSSION THREAD

**Discussion by mimhdev** [Budibase/budibase#12612] [Help] [answered]
Hi
I really appreciate your great work

I'm just looking for how to contribute/develop an alternate calendar/locale support for date, duration, etc.

Thanks

**Comment by ConorWebb96** [comment, marked as answer]
Hey @mimhdev,

You could create a [custom component](https://docs.budibase.com/docs/custom-component) for this alternative calendar.

I've linked the relevant documentation on how to do this above.
